tools/deploy_firmware.py

```diff
@@ -174,6 +174,10 @@ def upgrade_uboot(self):
             # make the shell prompt appear:
             self.set_prompt(shell)
             shell.expect(self.serial_prompt)
+            # kill any instance of the init script, if the current
+            # firmware doesn't have working wireless interfaces it
+            # will prevent it from rebooting:
+            shell.sendline("pgrep -f 'S99prplmesh boot' | xargs kill")
             # reboot:
             shell.sendline("reboot")
```

Comment:
It would be better to also do `reboot -f` here - which would have avoided the problem to begin with. There's really not much reason to go through the init scripts.

Reply:
Didn't know about the -f option

Reply:
I didn't know busybox's reboot had that. 
Thanks for the suggestion! 
I'll do that.